styles/style.css

```diff
@@ -0,0 +1,62 @@
+html {
+  min-height: 100vh;
+}
+
+body {
+  background-color: #f2f59a;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  min-height: 100vh;
```

Comment:
Most browsers add some margin or padding around the body by default. If you're going for the trim fit look where the body at least filsl the viewable window but adds no scrolling if not needed, try setting any vertical margin and padding to 0.